web/apps/labelstudio/src/pages/Projects/Projects.scss

```diff
@@ -13,6 +13,8 @@
   &__list {
     padding: 40px;
     display: grid;
+
+    // review in the pr this will fail in CI
```

Comment:
@yyassi-heartex we probably need to review this property that doesn't exist and also the value is not correct. It is part of the migration, should I just remove it?

Reply:
I'm pretty sure it was intended to be grid-auto-flow `https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow`
this has been in place since initial commit `Release 1.0.0`

Reply:
Let's remove the line